### ISSUE TITLE
Optimizer and Weight decay

### ISSUE THREAD

**Issue by nw89** [MichaelRamamonjisoa/SharpNet#5]
Regarding 6.4 in the paper, do you use actual weight decay or a simple L^2 regularisation term on the weights? Is the optimizer ordinary SGD or something like Adam?

**Comment by MichaelRamamonjisoa** [comment]
I used SGD optimizer with learning rate decay such that learning_rate = learning_rate * (1 - epoch /max_epoch) ^ 0.9,  weight_decay=2e-6 (typically) and momentum=0.9.

The optimizer I used is SGD.

**Comment by nw89** [comment]
Brilliant, thank you very much!
